crates/common/types/genesis.rs

```diff
@@ -200,6 +200,17 @@ pub struct ChainConfig {
     pub deposit_contract_address: Address,
 }
 
+lazy_static::lazy_static! {
+    pub static ref NETWORK_NAMES: HashMap<String, &'static str> = {
+        let mut m = HashMap::new();
+        m.insert(1.to_string(), "mainnet");
+        m.insert(11155111.to_string(), "sepolia");
+        m.insert(17000.to_string(), "holesky");
+        m.insert(560048.to_string(), "hoodi");
```

Comment:
Let's add the local dev networks: L1 with chain ID 9 and L2 with chain ID 65536999.